config.json

```diff
@@ -69,7 +69,8 @@
     },
     "travis": {
       "blacklist": [
-        "repos/polymer-starter-kit"
+        "repos/polymer-starter-kit",
+        "repos/paper-checkbox"
```

Comment:
@azakus now I'm trying this approach to use ESLint in elements: [IBMResearch/ibm-tabs/master/.travis.yml#L22-L23](https://github.com/IBMResearch/ibm-tabs/blob/master/.travis.yml#L22-L23). Maybe can help you.